templates/participant/details.html.twig

```diff
@@ -85,15 +88,129 @@
                         </li>
                     {% endfor %}
                 {% endif %}
-                {% if not isDVType %}
-                    <li role="presentation">
-                        <a href="#patientstatus" aria-controls="patientstatus" role="tab" data-toggle="tab">Patient Status</a>
-                    </li>
-                {% endif %}
             </ul>
             <br />
             <div class="tab-content">
-                <div role="tabpanel" class="tab-pane active" id="inperson">
+                <div role="tabpanel" class="tab-pane active" id="on_site_details">
+                    <div class="row">
+                        {% if not isDVType %}
+                            <div class="col-md-6">
+                                {% if canViewPatientStatus %}
```

Comment:
I like how you put the incentive panel in a separate view partial. Should we do the same for patient status?